debian/control

```diff
@@ -0,0 +1,33 @@
+Source: txfixtures
+Maintainer: Debian Python Modules Team <python-modules-team@lists.alioth.debian.org>
```

Comment:
Is this true?

Reply:
I will replace this and all the maintainer related details to Launchpad Engineering team.